src/Model/Tag.php

```diff
@@ -325,6 +325,32 @@ private static function getTypeForHash(string $hash)
 		}
 	}
 
+	/**
+	 * Create implicit mentions for a given post
+	 *
+	 * @param integer $uri_id
+	 * @param integer $parent_uri_id
+	 */
+	public static function createImplicitMentions(int $uri_id, int $parent_uri_id)
+	{
+		if ($uri_id == $parent_uri_id) {
+			return;
+		}
+
+		// Always mention the direct parent author
+		$parent = Item::selectFirst(['author-link', 'author-name'], ['uri-id' => $parent_uri_id]);
+		self::store($uri_id, self::IMPLICIT_MENTION, $parent['author-name'], $parent['author-link']);
+
+		if (DI::config()->get('system', 'disable_implicit_mentions')) {
+			return;
+		}
+
+		$tags = DBA::select('tag-view', ['name', 'url'], ['uri-id' => $parent_uri_id]);
+		while ($tag = DBA::fetch($tags)) {
```

Comment:
You seem to have forgotten the `DBA::close()` call. That's why I don't like this form. Even people advocating for it end up stumbling, and in practice there's no difference with `selectToArray` until we actually implement unbuffered queries.

Reply:
Or maybe I shouldn't watch too interesting Youtube videos while coding :-)

Reply:
But that's my point, you can watch Youtube videos while using `selectToArray`.